src/pages/ConnectWallet/ConnectWallet.tsx

```diff
@@ -86,10 +86,21 @@ export const ConnectWallet = ({ dispatch, hasWallet }: NoWalletProps) => {
           zIndex={1}
           colorScheme='blue'
           rightIcon={<ArrowForwardIcon />}
+          mb={8}
           onClick={() => dispatch({ type: WalletActions.SET_WALLET_MODAL, payload: true })}
         >
           <Text translation='connectWalletPage.cta' />
         </Button>
+
+        <Button
```

Comment:
The two CTAs are a different size because of the larger copy. To me as a user, it's a bit weird to have the alternative trial/ diminished experience without "signup" CTA being larger than the main one even though it's not showing the primary color. Maybe chakra's `ghost` variant would be a better choice?
  - current:  
  ![image](https://user-images.githubusercontent.com/17035424/152171838-8d7da764-03bd-4155-8ae5-b7c20bc396be.png)

  - ghost:
  ![image](https://user-images.githubusercontent.com/17035424/152171470-54622e19-fc2a-498d-8ee7-882e447e5b43.png)
  
  - outline:
  ![image](https://user-images.githubusercontent.com/17035424/152171594-8c1e9084-34fc-436c-ba7e-4ad950c3aa9a.png)

Reply:
ok, ill change it to the `ghost` variant